addons/mrp_account/tests/test_mrp_account.py

```diff
@@ -233,16 +233,17 @@ def test_unbuild_account_00(self):
         production = mo_form.save()
         production._post_inventory()
         production.button_mark_done()
-
+        production_in_account = self.product_A.property_stock_production.valuation_in_account_id
+        production_out_account = self.product_A.property_stock_production.valuation_out_account_id
         # finished product move
         productA_debit_line = self.env['account.move.line'].search([('ref', 'ilike', 'MO%Product A'), ('credit', '=', 0)])
         productA_credit_line = self.env['account.move.line'].search([('ref', 'ilike', 'MO%Product A'), ('debit', '=', 0)])
         self.assertEqual(productA_debit_line.account_id, self.stock_valuation_account)
-        self.assertEqual(productA_credit_line.account_id, self.stock_input_account)
+        self.assertEqual(productA_credit_line.account_id, production_out_account or self.stock_input_account)
```

Comment:
A test should only expect 1 deterministic end result value, i.e. you shouldn't be able to have either `production_out_account` or `self.stock_input_account`